site/en/docs/devtools/recorder/index.md

```diff
@@ -105,27 +113,55 @@ Let's walk through the options to edit the user flows.
      {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/g3x137aFK4xJrsdqONKY.png", alt="View all recordings", width="800", height="560" %}
-3. **Export a recording**. You can export the user flow as [Puppeteer](https://pptr.dev) script to customize the script further.
-4. **Delete a recording**. Delete the selected recording. 
+3. **Export a recording**<span class="material-icons">file_upload</span>. You can export the user flow as [Puppeteer](https://pptr.dev) script to customize the script further.
+4. **Delete a recording**<span class="material-icons">delete</span>. Delete the selected recording.
+
+You can also edit the recording's name by clicking the edit button <span class="material-icons">edit</span> next to it.
 
 ## Edit steps {: #edit-steps }
 
 Let's walk through the options to edit the steps within a workflow.
 
-1. Expand each step to see the details of the action. For example, expand the *Click Element "Cappucino"* step.
-  {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/ADbB4FFeiBBZqbxKRw0s.png", alt="In the recorder panel, the Cappucino element has been expanded to reveal type, target, selectors, offset X, and offset Y.", width="800", height="663" %}
-2. The step above shows 2 **selectors**. The **Recorder** will automatically detect 2 selectors for most of the steps by default.
-    - When replaying the user flow, the **Recorder** will try to query the element with one of the selectors by sequence.
-    - For example, if the **Recorder** successfully queries the element with the first selector, it will skip the second selector and proceed to the next step.
-3. You can add or remove any selectors. For example, you can remove the *selector #2* because just `aria/Cappucino` is sufficient in this case. Hover over the *selector #2* and click on **-** to remove it.
+### Expand steps {: #expand-step }
+
+1. Expand each step to see the details of the action. For example, expand the *Click Element "Cappuccino"* step.
+  {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/WUIZb8eMWfcPZHPyQ5C5.png", alt="In the recorder panel, the Cappuccino element has been expanded to reveal type, target, selectors, offset X, and offset Y.", width="800", height="773" %}
+1. The step above shows two **selectors**. For more information, see [Selector types](#selector-types).
+   When replaying the user flow, the **Recorder** tries to query the element with one of the selectors by sequence.
+   For example, if the **Recorder** successfully queries the element with the first selector, it will skip the second selector and proceed to the next step.
+1. You can add or remove any selectors. For example, you can remove the *selector #2* because just `aria/Cappuccino` is sufficient in this case. Hover over the *selector #2* and click on **-** to remove it.
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/qEat3jT1Og68urhmSkef.png", alt="The DevTools recorder panel shows an option to remove a selector", width="800", height="663" %}
```

Comment:
outdated image because the second selector is not using [data-test]. Mismaatch from the previous step